clients/src/main/java/org/apache/kafka/clients/consumer/internals/ShareConsumeRequestManager.java

```diff
@@ -1080,4 +1084,23 @@ public V getSyncRequest() {
     Pair<AcknowledgeRequestState> requestStates(int nodeId) {
         return acknowledgeRequestStates.get(nodeId);
     }
+
+    @InterfaceStability.Evolving
```

Comment:
You don't need to use `InterfaceStability` on an internal enum like this. That annotation is for public interface to Kafka to warn users that their code might have to change if the interface is still unstable (such as KafkaShareConsumer).

Reply:
Okay, I have removed it now.